<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_insertarorden</class>
 <widget class="QDialog" name="form_insertarorden">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insertar Orden</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Insertar Orden</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>No. de Orden</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="le_norder"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Fecha:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" colspan="2">
       <widget class="QDateEdit" name="de_fecha">
        <property name="currentSection">
         <enum>QDateTimeEdit::DaySection</enum>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QCheckBox" name="cb_pagado">
        <property name="text">
         <string>Pagado</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Plato:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="cb_plato">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Cantidad:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QDoubleSpinBox" name="sb_cantidad">
        <property name="maximum">
         <double>1000000000.990000009536743</double>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="pb_add">
        <property name="text">
         <string>AÃ±adir</string>
        </property>
        <property name="icon">
         <iconset resource="recursos.qrc">
          <normaloff>:/images/Images/iconos_colores/plus_96px.png</normaloff>:/images/Images/iconos_colores/plus_96px.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="6">
       <widget class="QTableWidget" name="tw_platos">
        <column>
         <property name="text">
          <string>Plato</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Cantidad</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Precio x Unidad</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Precio Total</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPushButton" name="pb_del">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="recursos.qrc">
             <normaloff>:/images/Images/iconos_colores/trash_can_96px.png</normaloff>:/images/Images/iconos_colores/trash_can_96px.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0" colspan="7">
       <widget class="QLabel" name="l_costo">
        <property name="text">
         <string>Costo de la Orden</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QPushButton" name="pb_accep">
        <property name="text">
         <string>Aceptar</string>
        </property>
        <property name="icon">
         <iconset resource="recursos.qrc">
          <normaloff>:/images/Images/iconos_colores/checked_96px.png</normaloff>:/images/Images/iconos_colores/checked_96px.png</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="6">
       <widget class="QPushButton" name="pb_cancel">
        <property name="text">
         <string>Cancelar</string>
        </property>
        <property name="icon">
         <iconset resource="recursos.qrc">
          <normaloff>:/images/Images/iconos_colores/cancel_96px.png</normaloff>:/images/Images/iconos_colores/cancel_96px.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
